.circleci/config.yml

```diff
@@ -116,6 +116,18 @@ jobs:
     steps:
       - run: yarn type-check
 
+  relay-check:
+    docker:
+      - image: 585031190124.dkr.ecr.us-east-1.amazonaws.com/force:$CIRCLE_SHA1-builder
+        aws_auth:
+          aws_access_key_id: $AWS_ACCESS_KEY_ID
+          aws_secret_access_key: $AWS_SECRET_ACCESS_KEY
+    working_directory: /app
+    environment:
+      - NODE_ENV: test
```

Comment:
Not sure if it matters for `yarn relay`, but I am curious if it makes more sense to be in dev environment instead of test for compiling rely.

Reply:
This section can likely just be removed

Reply:
by this section do you mean the `environment` block here?

Reply:
Yes, we can just remove the entire `environment` block.

Reply:
sounds good i removed!